scripts/enforce_header.py

```diff
@@ -2,9 +2,9 @@
 import os
 import sys
 
-HEADER_FILE = "scripts/header.py"
+# HEADER_FILE = "scripts/header.py"
 
-HEADER_SOURCE = open(HEADER_FILE).read().strip()
+# HEADER_SOURCE = open(HEADER_FILE).read().strip()
```

Comment:
Gah, this is what happens when I'm careless and don't double-check what I'm staging.